src/app/app.css

```diff
@@ -2,6 +2,8 @@ html, body, #root {
   height: 100%;
 }
 
-.pf-c-content small {
-  --pf-c-content--small--Color: red;
+.pf-c-content {
+  --pf-c-content--small--Color: red; /* changes all <small> color to red */
+  --pf-c-content--blockquote--BorderLeftColor: purple; /* changes all <blockquote> left border color to purple */
+  --pf-c-content--hr--BackgroundColor: lemonchiffon; /* changes a <hr> color to lemonchiffon */
```

Comment:
Cool! If this looks OK that works for me. It's a pretty faint yellow, I only used it in the comment because it's a fun word :) Other candidates might be `forestgreen`, `hotpink`, `tomato`. 

An alternative route could be to use our global var colors, color palette vars, and a string/hex color. Like `--pf-global--active-color--100` for the blockquote border color, or `--pf-global--danger-color--100` for the `small` color instead of `red`, or `--pf-global--palette--orange-100` for the `hr` background color.

Reply:
@mcoker I'd vote for what ever you think is most educational for anyone just starting with the react seed to learn as much as possible in the fewest lines :)

Reply:
Gotcha, a global var, palette var, and custom color covers a few bases on ways to override colors.

```
.pf-c-content {
  --pf-c-content--small--Color: var(--pf-global--danger-color--200);
  --pf-c-content--blockquote--BorderLeftColor: var(--pf-global--palette--orange-300);
  --pf-c-content--hr--BackgroundColor: red;
}
```